### ISSUE TITLE
fwd and rev unpaired reads are different sizes?

### ISSUE THREAD

**Issue by sipesk** [voutcn/megahit#162]
Hello,

I am using Megahit to assemble from paired, fwd and rev trimmomatic trimmed reads. For some reason, my forward and reverse reads aren't the same length, therefore giving me this error. How do I proceed?

barbara-lloydlab@lloydbarbara:~/megahit$ ./megahit -t 12 -1 /home/barbara-lloydlab/Desktop/KBaseKatie/New_binning_method/raw_reads/trimmomatic_trimmed/Kolyma35mTrimmed_unpaired_rev.fastq -2 /home/barbara-lloydlab/Desktop/KBaseKatie/New_binning_method/raw_reads/trimmomatic_trimmed/Kolyma35mTrimmed_unpaired_fwd.fastq -o 35m_Megahit
62.0Gb memory in total.
Using: 56.579Gb.
MEGAHIT v1.1.2
--- [Wed Apr 11 10:30:05 2018] Start assembly. Number of CPU threads 12 ---
--- [Wed Apr 11 10:30:05 2018] Available memory: 67501543424, used: 60751389081
--- [Wed Apr 11 10:30:05 2018] Converting reads to binaries ---
    [ERROR] [sequence_manager.cpp     : 151]: File(s) /home/barbara-lloydlab/Desktop/KBaseKatie/New_binning_method/raw_reads/trimmomatic_trimmed/Kolyma35mTrimmed_unpaired_rev.fastq,/home/barbara-lloydlab/Desktop/KBaseKatie/New_binning_method/raw_reads/trimmomatic_trimmed/Kolyma35mTrimmed_unpaired_fwd.fastq: Number of sequences not the same in paired files. Abort.
Error occurs when running "megahit_asm_core buildlib"; please refer to 35m_Megahit/log for detail
cat: write error: Broken pipe
[Exit code 1]

Thanks,
Katie

**Comment by sipesk** [comment]
Yes, that is intuitive, but how do I fix it so they are the same length?

**Comment by voutcn** [comment]
trimmomatic should have some option to let you separate output paired reads from orphan reads.
